### PR TITLE
Filter the custom loads done by the PCs.

### PR DESCRIPTION
**Pull request by melisabok**
Issue reported by Thomas.  PCs can see all the custom load notes so I'm filtering them by 'content.user'.

### CODE REVIEW

File: openreview/conference/templates/reviewerWebfield.js [hunk 1]
```diff
@@ -183,10 +183,17 @@ var displayHeader = function(headerP) {
 
   var reducedLoadP = $.Deferred().resolve(0);
   if (REVIEW_LOAD > 0) {
+    userIds = _.union([user.profile.id], user.profile.emails);
     reducedLoadP = Webfield.get('/notes', { invitation: CONFERENCE_ID + '/-/Reduced_Load'})
     .then(function(result) {
       if (result.notes && result.notes.length) {
-        return result.notes[0].content.reviewer_load;
+        if (result.notes.length == 1) {
```
Comment (zbialecki) [inline openreview/conference/templates/reviewerWebfield.js]: Should be ===

File: openreview/conference/templates/reviewerWebfield.js [hunk 2]
```diff
@@ -183,10 +183,17 @@ var displayHeader = function(headerP) {
 
   var reducedLoadP = $.Deferred().resolve(0);
   if (REVIEW_LOAD > 0) {
+    userIds = _.union([user.profile.id], user.profile.emails);
```
Comment (zbialecki) [inline openreview/conference/templates/reviewerWebfield.js]: Should you also include all ~ ids in the names array? Or are those never used for these notes?

File: openreview/conference/templates/reviewerWebfield.js [hunk 3]
```diff
@@ -183,10 +183,17 @@ var displayHeader = function(headerP) {
...
     reducedLoadP = Webfield.get('/notes', { invitation: CONFERENCE_ID + '/-/Reduced_Load'})
     .then(function(result) {
       if (result.notes && result.notes.length) {
-        return result.notes[0].content.reviewer_load;
+        if (result.notes.length == 1) {
+          return result.notes[0].content.reviewer_load;
+        } else {
+          //If there is more than one there might be a Program Chair
+          loads = _.filter(result.notes, function(note) { return userIds.indexOf(note.content.user) >= 0;});
```
Comment (zbialecki) [inline openreview/conference/templates/reviewerWebfield.js]: Missing `var` here. 

I think in this case we should just use the native filter, `result.notes.filter(...`

File: openreview/conference/templates/reviewerWebfield.js (hunk 2 above)
Comment (melisabok) [reply openreview/conference/templates/reviewerWebfield.js]: good catch! solved.

**Pull request merged**